src/PactNet.Abstractions/Matchers/ArrayContainsMatcher.cs

```diff
@@ -0,0 +1,17 @@
+using System.Text.Json.Serialization;
+
+namespace PactNet.Matchers;
```

Comment:
style: The project uses namespace blocks instead of file scoped namespaces

Reply:
Noted, will amend